brainglobe_atlasapi/atlas_generation/atlas_scripts/praire_vole.py

```diff
@@ -0,0 +1,256 @@
+__version__ = "0"
+
+import os
+import random
+import shutil
+import time
+import zipfile
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pooch
+import tifffile
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir):
+    ATLAS_NAME = "..."
```

Comment:
I've emailed the lab that originally made this atlas to see if they have a preference for the name, but for now it can just be `prairie_vole_brain`.